elliottlib/errata_async.py

```diff
@@ -1,16 +1,17 @@
 import asyncio
 import base64
-import logging
 from typing import Dict, Iterable, List, Set, Union
 from urllib.parse import quote, urlparse
 
 import aiohttp
 import gssapi
+import re
+import semver
 
 from elliottlib.rpm_utils import parse_nvr
-from elliottlib import constants
+from elliottlib import constants, util, logutil
 
-_LOGGER = logging.getLogger(__name__)
+_LOGGER = logutil.getLogger(__name__)
```

Comment:
This would let methods here print to our runtime's logger - instead of being lost in the void